### DISCUSSION TITLE
Switch to inclusive terms in the Godot Engine and documentation

### DISCUSSION THREAD

**Discussion by GreenHapi** [godotengine/godot-proposals#7959] [General Discussions]
https://docs.godotengine.org/en/stable/search.html?q=master&check_keywords=yes&area=default#

For example, the `master` is an outdated term. Master -> Main:
https://docs.godotengine.org/en/stable/tutorials/2d/2d_transforms.html#global-canvas-transform
`This is the master transform and affects all individual Canvas Layer transforms.`
can be
`This is the main transform and affects all individual Canvas Layer transforms.`

`Master Bus` can be `Main Bus`
https://docs.godotengine.org/en/stable/tutorials/audio/audio_buses.html

For reference:
https://developers.google.com/style/word-list#master
https://developers.google.com/style/inclusive-documentation
https://helpx.adobe.com/x-productkb/multi/terminology-changes-video-products.html

**Comment by Broken1334** [comment]
I think that "master" is ok, since there is no "slave" combination.

**Comment by akien-mga** [comment]
The use of "master" in that documentation paragraph can easily be replaced by an alternative, I agree.

For "Master Bus", we evaluated this when we removed the master/slave terminology from the networking API years ago, and we chose to keep the audio one as is. It's the canonical name for what it's representing in all audio software and hardware, so coming up with our own alternative when professional audio software doesn't seems like it would be confusing to users. AFAIK the term derives from [mastering](https://en.wikipedia.org/wiki/Mastering_(audio)), which is also used in widespread terms like "remaster".

If there's evidence of audio software adopting an alternative term for this, then we can definitely follow suit.

**Reply by GreenHapi** [reply]
There is. I attached a link to the Adobe page in the post above.
https://helpx.adobe.com/x-productkb/multi/terminology-changes-video-products.html
Here's an excerpt.
```
Terminology change in Audition
In Audition v14.0 and later

Master Audio Track will be called Mix Audio Track and Master Track is now Mix Track.
Master Template in Audition will be called Template.
```

**Comment by Mickeon** [comment]
For anything audio-related, changing away from "Master" would  break compatibility so it would have to be done way, way later. But should it be done? It'd be more confusing than it is worth, in my opinion. It would feel like changing for the sake of being different. In the audio department the term is too common to let go, and has basically nothing to do with with the concept of master/slave.

**Reply by GreenHapi** [reply]
```
Audio terms such as “master and slave” and “male/female connectors” could be made obsolete soon,
thanks to a push for neutral and inclusive language by the Professional Audio Manufacturers Alliance (PAMA).
```

```
PAMA’s members include some of the biggest names in professional audio,
including Audio-Technica, Audix, Harman, Sennheiser and Shure.
```
https://musictech.com/news/industry/pama-initiative-inclusive-neutral-audio-terminology/

**Reply by Mickeon** [reply]
I get it. I hope the efforts do not go in vain, but this still needs to be remembered for the far future
